api/environments/tests/test_views.py

```diff
@@ -536,6 +539,53 @@ def test_create_environment_without_required_metadata_returns_400(
     assert "Missing required metadata field" in response.json()["metadata"][0]
 
 
+def test_view_environment_with_staff_track_query_count(
+    staff_client: FFAdminUser,
+    environment: Environment,
+    with_environment_permissions: Callable,
```

Comment:
nit: we should type this properly. Looks like it should be

```suggestion
    with_environment_permissions: Callable[[list[str], int], None],
```

I can see the fixture definition is also not typed correctly.